include/nvexec/detail/queue.cuh

```diff
@@ -101,7 +101,8 @@ namespace nvexec::STDEXEC_STREAM_DETAIL_NS {
     struct producer_t {
       task_base_t** tail_;
 
-      void operator()(task_base_t* task) {
+      STDEXEC_DETAIL_CUDACC_HOST_DEVICE //
+      inline void operator()(task_base_t* task) {
```

Comment:
I wonder why `inline` is necessary here. Isn't it implicit for member functions defined _in situ_?